lib/reporting/drop_off_report.rb

```diff
@@ -223,15 +223,15 @@ def dropoff_metrics_table
         ],
         [
           'Verified (event)',
-          idv_personal_key_submitted,
```

Comment:
if `perosnal_key_submitted` is no longer used, should we remove the method to calculate it + remove it from the events that are queried from CW to reduce the amount of data being pulled?

Reply:
I'll check with Product on this one.

Reply:
Looking at this more closely, we have the analytics event we sent to CW, and then this method on `Reporting::DropOffReport`. 
Do report methods optimistically run? If I left the method and it's not called, is the data being pulled?

Reply:
The constant is defined here:
https://github.com/18F/identity-idp/blob/f63db2f8f639655db091c655d39a67745e0c648c/lib/reporting/drop_off_report.rb#L50

so it will be included in `all_events` here:
https://github.com/18F/identity-idp/blob/f63db2f8f639655db091c655d39a67745e0c648c/lib/reporting/drop_off_report.rb#L54-L56

so it's quoted as an array here:
https://github.com/18F/identity-idp/blob/f63db2f8f639655db091c655d39a67745e0c648c/lib/reporting/drop_off_report.rb#L400

and used as a `filter name IN ...` here:

https://github.com/18F/identity-idp/blob/f63db2f8f639655db091c655d39a67745e0c648c/lib/reporting/drop_off_report.rb#L414

so yes, removing the constant will lessen the amount of data queried

Reply:
plus no point in leaving around dead/unused code in general, that just makes it harder for future folks to understand what's going on if something is partially removed

Reply:
ok, excellent. I'm figuring out how it all fits together. 
Thank you!